### ISSUE TITLE
Hello?

### ISSUE THREAD

**Issue by tekedot21** [atomic-software/TISFAT-Zero#82]
Is there anything new going on? It seems like this has been abandoned. 

Side note: waiting for ffmpeg is endless. I cannot export to any format.

**Comment by desmondcayce** [comment]
Looks pretty abandoned to me...

**Comment by qoh** [comment]
It isn’t maintained, no.

cc/ @evsl

**Comment by evsl** [comment]
Indeed. Worth noting that this issue itself is a good few years old as well @desmondcayce 

If people are interested in tisfat still, then that could change. There are several much more mature versions in currently non-public states that I would find interesting to work on again.
